3rdparty/SConscript

```diff
@@ -319,6 +319,50 @@ elif 'sox' in system_dependencies:
 
     env = conf.Finish()
 
+# dep: sndfile
+if 'sndfile' in autobuild_dependencies:
+    sndfile_deps = []
+
+    if 'alsa' in autobuild_dependencies:
+        sndfile_deps += ['alsa']
+
+    if 'pulseaudio' in autobuild_dependencies:
+        sndfile_deps += ['pulseaudio']
+
+    env.BuildThirdParty(thirdparty_versions, 'sndfile', deps=sndfile_deps)
```

Comment:
I think libsndfile does not depend on ALSA and pulseaudio?

Actually it's reverse: pulseaudio depends on libsndfile.